setup.py

```diff
@@ -13,6 +13,8 @@
       install_requires=[
           "PyYAML",
       ],
+      setup_requires=['setuptools_scm'],
```

Comment:
without this change, the `third_party` directory wasn't getting included during installation